composer/soso-richtext.html

```diff
@@ -366,7 +367,7 @@
 
         // remove ignored nodes
         const ignoreList = ["figure", "img", "image", "form", "input", "select", "video", "audio", "meta", "script", "link", "meta", "applet", "aside",
-          "object", "entity", "button", "canvas", "svg", "font", "frame", "iframe"];
+          "object", "entity", "button", "canvas", "svg", "font", "frame", "iframe", "style"];
```

Comment:
What about "class"?

Reply:
These are tag names not attribute names

Reply:
Oh, yeah.  Do we need to strip any attributes?

Reply:
Already being done few lines down in the code. Removing all 'style' and 'class' attributes. Style could also be a node which may add general formatting rules.